graph/src/data/store/mod.rs

```diff
@@ -164,21 +165,14 @@ impl Value {
                     .map(|value| Self::from_query_value(value, &NamedType(n.to_string())))
                     .collect::<Result<Vec<_>, _>>()?,
             ),
-            (query::Value::Enum(e), NamedType(n)) => {
-                // Check if `ty` is a custom scalar type, otherwise assume it's
-                // just a string.
-                match n.as_str() {
-                    BYTES_SCALAR => Value::Bytes(scalar::Bytes::from_str(e)?),
-                    BIG_INT_SCALAR => Value::BigInt(scalar::BigInt::from_str(e)?),
-                    _ => Value::String(e.clone()),
-                }
-            }
+            (query::Value::Enum(e), NamedType(_)) => Value::String(e.clone()),
```

Comment:
Interesting, the above didn't really make sense, huh (an enum value that represents a `Bytes` or `BigInt` value isn't even possible, since enum values don't have quotes around them like strings/bytes/bigints)?

Reply:
Yhea I understand that an enum is an unquoted string, and those can never be numbers, so it doesn't seem to make sense.